substrate/service/src/lib.rs

```diff
@@ -50,7 +50,7 @@ extern crate serde_derive;
 
 mod components;
 mod error;
-mod config;
+pub mod config;
```

Comment:
could you group the public module declarations together?

Reply:
Yes, ok.

Reply:
New change committed to group the public module declarations together.